src/test/java/red/softn/npedidos/controller/OrderControllerTestUtil.java

```diff
@@ -0,0 +1,27 @@
+package red.softn.npedidos.controller;
+
+import com.google.gson.Gson;
+import lombok.Getter;
+import red.softn.npedidos.ControllerTestUtil;
+import red.softn.npedidos.TestUtil;
+import red.softn.npedidos.request.OrderRequest;
+import red.softn.npedidos.response.OrderResponse;
+
+@Getter
+public class OrderControllerTestUtil extends ControllerTestUtil<OrderRequest, OrderResponse, Integer> {
+    
+    public OrderControllerTestUtil(Gson gson) {
+        super(gson);
+    }
+    
+    @Override
+    protected Init<OrderRequest, OrderResponse, Integer> init() {
+        var request = new OrderRequest();
+        var response = new OrderResponse();
+        request.setId(TestUtil.fakeRandomInteger());
+        response.setId(request.getId());

```

Comment:
property `dateOrder` is missing

Reply:
@nmarulo ,
Thanks for the feedback. dateOrder property is added for both Request and Response objects.